### PR TITLE
Fixing misc. bugs

### PR DESCRIPTION
**Pull request by fosskers**
**Note:** This is based on the `use-histograms` branch (#26). That should be merged first.

### Bugs
- [x] Layer flickering upon load
- [x] Metadata not displaying until zoomed once
- [ ] ~~Metadata being fetched twice per zoom~~ (not every time, and not a big deal)
- [x] Show current settings in `Map Settings`

### CODE REVIEW

**Comment by fosskers** [comment]
The initial state of the circuit:

circuit

**Comment by fosskers** [comment]
After flicker fix:

circuit2

File: client/src/main/scala/geotrellis/admin/client/components/modal/LayerList.scala [hunk 1]
```diff
@@ -16,71 +18,55 @@ import japgolly.scalajs.react.vdom.prefix_<^._
...
+          <.div({
+            val layDs: Seq[LayerDescription] = layers.sortBy(_.name)
+
+            val options: Seq[ValueOption[ReactNode]] = layDs.zipWithIndex.map { case (option, index) =>
+              ValueOption[ReactNode](value = index, label = option.name)
+            }
+
+            // For some reason, `.getOrElse` won't compile here.
+            val sel: JsCol[ReactNode] = proxy().selection.flatMap(ld =>
+              options.filter(_.label == ld.name).headOption.map(_.value)  // Naive.
```
Comment (fosskers) [inline client/src/main/scala/geotrellis/admin/client/components/modal/LayerList.scala]: In Haskell, the default List type is in a Scala sense a lazy, immutable `Stream`. It's a typical idiom to do `head . filter`, knowing that the filter will stop as soon as a `head` is found. Would a Scala `Stream` do that?

The `Seq`s here are actually `Array`. Perhaps the total layer lists aren't long enough to worry about optimizing this.

Comment (moradology) [inline client/src/main/scala/geotrellis/admin/client/components/modal/LayerList.scala]: Streams are used exactly as you describe. In practice though, I don't think the pattern is all that common

**Comment by moradology** [comment]
+1 Cleanliness: much improved

**Pull request merged**